main.js

```diff
@@ -1,28 +1,33 @@
-require("dotenv").config()
-const express = require("express")
-const cors = require("cors")
-const bodyParser = require("body-parser")
+require("dotenv").config();
```

Comment:
linter ce poludeti od tackizareza koje je tvoj editor dodao, ali sredicemo to naknadno, nije neka greska

Reply:
Na poslu me zeza linter, na laptopu cu ispeglati :)